cat/CaT-service.yaml

```diff
@@ -3296,7 +3296,32 @@ components:
               type: string
               format: date-time
               description: Date/time - Event Created
-
+            buyerQuestions:
+              type: array
+              items:
+                allOf:
+                  - $ref: '#/components/schemas/EvalCriteria'
+                  - type: object
+                    properties:
+                      relatesTo:
+                        type: string
+                      relateItems:
+                        type: string
+                      requirementGroups:
+                        allOf:
+                          - $ref: '#/components/schemas/QuestionGroup'
+                          - type: object
+                            properties:
+                              OCDS:
+                                type: object
+                                properties:
+                                  requirements:
```

Comment:
Similar comment as above - should this maybe be `questions` (as `requirement` is a different schema object)?

Reply:
Unfortunately it is defined in OCDS as requirements

Reply:
Should it be of type `Requirement` in that case if it is OCDS? As `Question` contains `Requirement` and also non OCDS fields?

Reply:
Hmmm, trying to think. I think I have got myself into a bit of a mess by conflating questions (requirements) and question generation

Reply:
In OCDS requirement=question. The problem is we have found items outside OCDS and put them in a "nonOCDS" section at each level. This is OK for when they are separate endpoints but when you glue them all together in one structure it becomes this mess. Maybe have a chat when you are free.